entities.py

```diff
@@ -13,15 +15,40 @@ def __init__(self, name, pos, type):
         self.pos = pos
         self.type = type
 
+    @abstractmethod
+    def move(self, direction): pass
```

Comment:
Why not implement it here instead of making it abstract? After all, every entity is probably going to implement it the same way, like in Actor.

Reply:
ok